build/scripts/EnlistmentHelperFunctions.psm1

```diff
@@ -206,7 +206,15 @@ function Get-PackageLatestVersion() {
                 }
             }
 
-            return Get-LatestBCArtifactVersion -minimumVersion $minimumVersion
+            $currentBranch = Get-CurrentBranch
+            $storageAccountOrder = @("bcartifacts", "bcinsider")
+            if($currentBranch -eq "main") {
```

Comment:
Shouldn't we always use bcinsider for .x branches well?

Reply:
No, actually, most of the time we should use _bcartifacts_ for baselines for .x branches. Only around major branching, when the .0 branch isn't shipped yet, we'll need to fallback to _bcinsider_.

Reply:
Ah yes you're right.. I was thinking of artifact versions